package-lock.json

```diff
@@ -1,8 +1,2638 @@
 {
```

Comment:
this package-lock doesn't seem right, it  contain the wrong name and version and also adds ridiculous amount of lines. Isn't that caused by some link or something? can we cleanup the changes here?

Reply:
hmm 🤔 that is true. I will check and clean it up